docs/UserGuide.md

```diff
@@ -127,7 +127,7 @@ Format: `list`
 
 Modify and/or updates existing policy information in the database.
 
-Format: `edit INDEX [n/NAME] [i/NRIC] [p/CONTACT NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG] [c/COMPANY] [l/LICENCE PLATE]
+Format: `edit INDEX [n/NAME] [i/NRIC] [p/CONTACT NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]... [c/COMPANY] [l/LICENCE PLATE]
```

Comment:
This will provide clarify to the users that multiple tags can be specified, nice!